### PR TITLE
[4.0] Joomlaupdate quickicon js typos

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue #22020

### Summary of Changes
Normalised js on the model of extensionupdate.

Can't be tested with patchtester!

### Testing Instructions

See #22020

### After patch
screen shot 2018-09-06 at 12 24 27

screen shot 2018-09-06 at 12 24 45

Not perfect as there may be other errors, due to the update pack or not, but now readable.

### CODE REVIEW

File: build/media_src/plg_quickicon_joomlaupdate/js/jupdatecheck.es6.js [hunk 1]
```diff
@@ -34,9 +34,7 @@
             if (updateInfo.version !== options.version) {
               const messages = {
                 message: [
-                  `Joomla.JText._('PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_MESSAGE').replace("%s", '<span class="badge badge-light">${updateInfoList.length}</span>')`
-                  + `<button class="btn btn-primary" onclick="document.location='${options.url}'">`
-                  + `${Joomla.JText._('PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_BUTOON')}</button>`,
+                  `${Joomla.JText._('PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_MESSAGE').replace('%s', `<span class="badge badge-pill badge-danger">${updateInfoList.length}</span>`)}<button class="btn btn-primary" onclick="document.location='${options.url}'">${Joomla.JText._('PLG_QUICKICON_EXTENSIONUPDATE_UPDATEFOUND_BUTTON')}</button>`,
```
Comment (brianteeman) [inline build/media_src/plg_quickicon_joomlaupdate/js/jupdatecheck.es6.js]: can you  please put it on multiple lines as it was before - it makes it much much more readable

**Comment by infograf768** [comment]
done

**Comment by infograf768** [comment]
drone failure unrelated

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on ad19714191c0374ea30f1d942df0ed51e3329548<br /><br />Can be tested with patchtester if doing an "npm install" afterwards.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22023">issues.joomla.org/tracker/joomla-cms/22023</a>.</sub>

**Comment by PhocaCz** [comment]
Hi,
tested now, everything is OK (tested successfully):

Imgur

> Can be tested with patchtester if doing an "npm install" afterwards.

Confirmed, I did the same way.

:white_check_mark:

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Pull request merged**